package.json

```diff
@@ -16,7 +16,7 @@
         "abi-parser": "./tooling/abi-parser.js"
     },
     "scripts": {
-        "compile": "truffle compile",
+        "compile": "npx truffle compile",
```

Comment:
we don't need `npx` (or `npm`) here because the node package module understands that it will need to run that command based on the dependencies in the project (and not a global installation)

---

We do need it on the other scripts because those are not exported binaries of another package we depend on